.goreleaser.yml

```diff
@@ -6,26 +7,53 @@ before:
 builds:
   - env:
       - CGO_ENABLED=0
-    ldflags:
-     - "-X=main.version={{.Env.VERSION}}"
     goos:
       - linux
       - windows
       - darwin
 archives:
-  - replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+  - format: tar.gz
+    # this name template makes the OS and Arch compatible with the results of `uname`.
+    name_template: >-
+      "Bitbucket CLI {{  .Version }}"_
+      {{- title .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else if eq .Arch "386" }}i386
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
+    # use zip for windows archives
+    format_overrides:
+      - goos: windows
+        format: zip
+
 checksum:
   name_template: 'checksums.txt'
 snapshot:
-  name_template: "{{ .Tag }}-next"
+  version_template: "{{ .Tag }}-next"
 changelog:
   sort: asc
   filters:
     exclude:
       - test
       - README
+
+nfpms:
+  - license: MIT
+    maintainer: 'Ivaaaan hello@iillexial.me'
+    homepage: https://github.com/ivaaaan/smug
+    dependencies:
+      - git
+    description: Bitbucket client command line tool.
```

Comment:
Can you change this to `Session manager and task runner for tmux. Start your development environment within one command.` please?

Reply:
sorry didn't mean to include these files in this PR I will update it